pkg/client/metadata.go

```diff
@@ -29,7 +29,7 @@ const classicEnvironmentDomainPath = "/platform/metadata/v1/classic-environment-
 const deprecatedClassicEnvDomainPath = "/platform/core/v1/environment-api-info"
 
 type classicEnvURL struct {
-	Endpoint string `json:"endpoint"`
+	Domain string `json:"domain"`
```

Comment:
With what version does that change? 
And I guess it doesn't change for both the new and the depracted endpoint? 

I think we'd be best of, if we tried to read both Domain and Endpoint in the JSON and take what's set

Reply:
done

Reply:
it's covered in TestGetDynatraceClassicEnvironmentFallsBackToDeprecatedPath